src/scss/components/selector-strip/_index.scss

```diff
@@ -0,0 +1,127 @@
+@import "vars";
+@import "mixins/focus";
+@import "components/form/mixins";
+
```

Comment:
Migrated Selector Strip styles and converted to SCSS

This file also verifies that mixins are working as expected.